lib/features/composer/presentation/composer_view.dart

```diff
@@ -109,68 +115,43 @@ class ComposerView extends GetWidget<ComposerController> {
       child: Container(
         margin: EdgeInsets.zero,
         color: AppColor.bgComposer,
-        padding: EdgeInsets.only(top: 24, bottom: 30, left: 18, right: 18),
+        padding: EdgeInsets.only(bottom: 30),
         alignment: Alignment.topCenter,
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          crossAxisAlignment: CrossAxisAlignment.start,
-          mainAxisSize: MainAxisSize.max,
-          children: [
-            (TextFormFieldBuilder()
-                ..key(Key('email_body_input'))
-                ..textInputAction(TextInputAction.newline)
-                ..textStyle(TextStyle(color: AppColor.nameUserColor, fontSize: 14))
-                ..textDecoration(InputDecoration(
-                    hintText: AppLocalizations.of(context).hint_content_email_composer,
-                    hintStyle: TextStyle(color: AppColor.baseTextColor, fontSize: 14),
-                    contentPadding: EdgeInsets.only(bottom: 20, left: 10, right: 10),
-                    filled: true,
-                    border: InputBorder.none,
-                    fillColor: AppColor.bgComposer)))
-              .build(),
-            _buildHeaderEmailsQuoted(context),
-            _buildContentEmailsQuoted(context),
-          ],
+        child: Obx(() => controller.getEmailActionType() == EmailActionType.createNew
+            ? _buildEmailBodyEditor(context)
+            : _buildEmailBodyEditorQuoted(context)
         )
       )
     );
   }
 
-  Widget _buildHeaderEmailsQuoted(BuildContext context) {
-    return Obx(() {
-      final localeCurrent = '${Localizations.localeOf(context).languageCode}_${Localizations.localeOf(context).countryCode}';
-      final headerEmailQuoted = controller.getHeaderEmailQuoted(localeCurrent);
-      return  headerEmailQuoted != null
-        ? Padding(
-            padding: EdgeInsets.symmetric(vertical: 10),
-            child: Text(
-              AppLocalizations.of(context).header_email_quoted(headerEmailQuoted.value1, headerEmailQuoted.value2),
-              style: TextStyle(color: AppColor.nameUserColor, fontSize: 14, fontStyle: FontStyle.italic),
-            ))
-        : SizedBox.shrink();
-    });
-  }
+ Widget _buildEmailBodyEditor(BuildContext context) {
+    return (TextFormFieldBuilder()
+        ..key(Key('email_body_editor'))
+        ..textInputAction(TextInputAction.newline)
+        ..textStyle(TextStyle(color: AppColor.nameUserColor, fontSize: 14))
+        ..addOnTapActionClick(() => keyboardUtils.hideKeyboard(context))
+        ..textDecoration(InputDecoration(
+            hintText: AppLocalizations.of(context).hint_content_email_composer,
+            hintStyle: TextStyle(color: AppColor.baseTextColor, fontSize: 14),
+            contentPadding: EdgeInsets.only(bottom: 24, left: 28, top: 24, right: 28),
+            filled: true,
+            border: InputBorder.none,
+            fillColor: AppColor.bgComposer)))
```

Comment:
why not use `HtmlEditor` also, because in the future, we will support rich text for composer also?